File: test_data/java_programs/openjdk_testsuite/01012.java




class InnerMemberRegression {
    void  test() {
        class Z {
            class Y {
                void test() {
                }
            }
            Y y = new Y();
            int m=100;

            void test() {
                y.test();
            }
        }  
        Z z = new Z();
        z.test();
    }

    public  static  void  main(String[]  s) {
        InnerMemberRegression x  =  new InnerMemberRegression();
        x.test();
    }
}

File: test_data/c_programs/clang_testsuite/01012.c
struct test {
   signed char a;
};
char c;
struct test s;
extern char ec;
extern struct test es;
void func (void)
{
  c = ec;
  s = es;
}
struct arg { long y __attribute__((packed, aligned(4))); };
extern struct arg x;
void f(struct arg);
void test (void)
{
  f(x);
}
